README.md

```diff
@@ -104,7 +104,7 @@ Options:
       --version  Show version number                                                                                                                                          [boolean]
   -v, --verbose  Run with verbose logging                                                                                                                                     [boolean]
   -n, --network  The network to use                                                                                                                         [string] [default: "testnet"]
-  -a, --account  The account to use                                                                                                                              [string] [default: ""]
+  -a, --accountIndex  The account to index referring to the account to use                                                                                                                              [string] [default: ""]
```

Comment:
```suggestion
  -a, --accountIndex  The index of the account to use                                                                                                                              [string] [default: ""]
```

Reply:
should this be a number instead of string? also personally I think it's more intuitive to use the address